client/filter/tvs.density.js

```diff
@@ -122,6 +114,26 @@ function applyBrush(self, elem, brush) {
 		)
 }
 
+export function updateTempRanges(xscale, s, range, inputRange, minvalue, maxvalue, type) {
+	range.start = convertRangeValue(xscale, s[0])
+	range.stop = convertRangeValue(xscale, s[1])
+	const min = roundValueAuto(Number(minvalue))
+	const max = roundValueAuto(Number(maxvalue))
+	//Limit by the brush, not by the user
+	range.startunbounded = min == range.start && inputRange.startunbounded
+	range.stopunbounded = max == range.stop && inputRange.stopunbounded
+	//Do not show decimals for integer types
+	if (!range.startunbounded && !range.stopunbounded && type == 'integer') {
```

Comment:
this means when range is one side unbounded, it won't handle integer case?

Reply:
Oh, I see what you mean. Thanks for pointing that out.